charts/private/portfolio/values.yaml

```
@@ -33,3 +33,6 @@ base-template:
     periodSeconds: 300
     timeoutSeconds: 3
     failureThreshold: 1
+github:
+  username: <GITHUB_USERNAME> # Required: The GitHub account username.
+  apiToken: <GITHUB_API_TOKEN> # Required: The GitHub API token.
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)